lib/va_profile/v2/contact_information/transaction_response.rb

```diff
@@ -22,7 +22,6 @@ class TransactionResponse < VAProfile::Response
           zip_code5
           zip_code4
           phone_number
-          country_code_iso3
```

Comment:
Question: does this change mean that the country code will no longer be redacted in the case that it is not `null`? Or is this field something that is not used anymore?
If there was more info in the error you linked I apologize, nothing shows on DataDog when I click on that link.

Reply:
@Diana20920 PII was removed from the transaction error logs. country_code_iso3 is null and no reason to redact the field. [Staging logs](https://vagov.ddog-gov.com/logs?query=env%3Aeks-staging%20service%3Avets-api%20%22undefined%20method%20%60country_code_iso3%27%22&agg_m=count&agg_m_source=base&agg_t=count&clustering_pattern_field_path=message&cols=host%2Cservice%2C%40http.referer%2C%40named_tags.dd.version&fromUser=true&graphType=flamegraph&messageDisplay=inline&refresh_mode=sliding&sort=time&spanID=1916012904192703251&storage=hot&stream_sort=time%2Casc&viz=stream&from_ts=1745947312550&to_ts=1747156912550&live=true)